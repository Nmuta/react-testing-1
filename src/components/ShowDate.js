import React from 'react'

const ShowDate = () => (
  <div>
    {/*<span>
      The correct date is: { new Date().toString() }
    </span>*/}

    {
    <span>
      The correct date is: 2017-04-05
    </span>
    }
  </div>
)

export default ShowDate