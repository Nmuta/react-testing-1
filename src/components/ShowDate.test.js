import React from 'react';
import ShowDate from './ShowDate'
import renderer from 'react-test-renderer';

test('renders the same way every time', () => {
  const tree = renderer.create( <ShowDate /> ).toJSON();
  expect(tree).toMatchSnapshot();
});