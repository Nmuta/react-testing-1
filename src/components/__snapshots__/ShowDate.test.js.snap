// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the same way every time 1`] = `
<div>
  <span>
    The correct date is: 2017-04-05
  </span>
</div>
`;
